src/problems/triangle_LSJZ.clj

```diff
@@ -0,0 +1,92 @@
+(ns problems.triangle-LSJZ)
+
+; Solve the triangle puzzlie with search.
+
+
+;; Example of creating states
+;; start-state (tri/->State [[1] [1 1] [1 1 1] [1 1 1 1] [1 1 1 1 0]])
+;; goal-state (tri/->State [[0] [0 0] [1 0 0] [0 0 0 0] [0 0 0 0 0]])
+
+(defrecord State [board])
+
+(defn get-all-coords [state]
+   (apply concat
+          (map
```

Comment:
When you have `(apply concat (map …))` (which appears down below as well, you can replace that with a call to `mapcat`.